utils/utils.js

```diff
@@ -105,17 +103,18 @@ async function createDocumentHandler() {
 			docCompletionButton: document.getElementById('docCompletionButton').value || 'Complete DBQ'
 		}
 		
-		const doc = await zegal.createDocument(doctypePayload, options)
+		const doc = await zegal.createDocument(doctypePayload, options, 'modalBodyId')
```

Comment:
this value is still hardcoded here.
Are you not suppose to get it back as parameter from somewhere ?

Reply:
Taking API portal as 3rd party client, this is dev who is sending their modal div-id, where we will show our application. Previously zegal-client had that hard coded; now 3rd party dev sends their div id @Anthony-Michel

Reply:
This was my understanding when HC told, 3rd party dev needs to set id not us. Please correct me if I am wrong
@Anthony-Michel